### PR TITLE
Fix note for Vbox users and fix indentations

### PR DESCRIPTION
**Pull request by bexelbie**

### CODE REVIEW

**Comment by jzb** [comment]
So - technically, we *are* producing VirtualBox images, but they're for Vagrant. I wonder if we'd be better off explaining how to strip the .box file to get the vbox image?

**Comment by nzwulfin** [comment]
If we're producing an image for Vagrant, how tough would it be to publish a "bare" VirtualBox image?

**Comment by jzb** [comment]
That's a question for Fedora rel-eng, the Fedora cloud group, and CentOS - respectively. I'm guessing technically, not horribly horribly difficult. Process-wise, medium difficult.

The big question that comes up here would be testing. Each image requires separate testing. Not sure if testing Vagrant for vbox counts as testing an independent VirtualBox image.

It also raises the question of whether a standalone VirtualBox image is at all strategic for the project.

**Comment by nzwulfin** [comment]
True, I think the VirtualBox image is really for Vagrant support as that was the most common combination we found.  I don't think a standalone image is valuable enough at this point to chase down.

So what does it take to strip a .box file for those folks who don't have qemu installed?  Having both options is probably a good idea.

**Comment by jzb** [comment]
It's dead easy. The box format is just a tar archive. tar -xvf will strip the file into a box.ovf, Vagrantfile, .vmdk file, and a metadata.json file. The ovf and .vmdk are all you need, I think.

**Comment by jberkus** [comment]
@bexelbie going through the backlog of PRs for Atomic-Site.  Is there a reason you're adding backticks to the command?

**Comment by nzwulfin** [comment]
Formatting note, Middleman will render these two code blocks differently (and differently than Github will render them).  

The triple tick creates a large dark grey box around the command the width of the column with an extra line at the bottom.  The indentation level affects the position of the box and the text separately.  This is the same box created by an indent only code block.

image

The single tick creates a small light grey box that only surrounds the text.  This allows for a code block embedded in a line or as a standalone line.

image

We don't have a style guide for documentation that says where either is appropriate, but it's important to point out so the changes are expected.

**Comment by jberkus** [comment]
@nzwulfin ... except, when I build your change, that's not how it appears.  I see the backticks in the doc page, not a grey box.

File: source/docs/quickstart.md [hunk 1]
```diff
@@ -8,12 +8,9 @@ We recommend reading the [Getting Started Guide](http://www.projectatomic.io/doc
...
 * **A virtual machine image.** Images for Atomic hosts are produced by both the Fedora Project and the CentOS Project.  Downloads for these images can be found via the [Downloads page] (http://www.projectatomic.io/download/) in QCOW2, RAW, and Vagrant BOX formats.
 
-* **Note for VirtualBox users** We are not producing native VirtualBox images, but you can generate your own VirtualBox image from the qcow2 images with `qemu-img`:
+  **Note for VirtualBox users** We are not producing native VirtualBox images, but you can generate your own VirtualBox image from the qcow2 images with `qemu-img`:
 
-````
-$ qemu-img convert -f qcow2 [filename].qcow2 -O vdi [filename].vdi
-
-````
+    `$ qemu-img convert -f qcow2 [filename].qcow2 -O vdi [filename].vdi`
```
Comment (nzwulfin) [inline source/docs/quickstart.md]: The indentation of the '`qemu` line was set to match the bulleted list indent, both lines will need to move to the start of the line to match indent levels.

**Comment by bexelbie** [comment]
I've updated the indention levels and I think the desired effect is now achieved.

**Comment by jberkus** [comment]
Looks good to me.  Merging.

**Pull request merged**